budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/persistence/budget/BudgetEntity.java

```diff
@@ -19,7 +19,8 @@
 @Entity
 @Table(name = "BUDGET",
         uniqueConstraints = {
-                @UniqueConstraint(name = "UNIQUE_IMPORT_KEY_PER_PROJECT", columnNames = {"PROJECT_ID", "IMPORT_KEY"})
+                @UniqueConstraint(name = "UNIQUE_IMPORT_KEY_PER_PROJECT", columnNames = {"PROJECT_ID", "IMPORT_KEY"}),
```

Comment:
This commit is missing a FlyWay script which does the database update on existing schemes.

Reply:
Added a migration script